dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/metrics/MetricsServiceBean.java

```diff
@@ -179,6 +179,34 @@ public long dataversesPastDays(int days) throws Exception {
                 .getResultList());
     }
 
+    /**
+     * Downloaded Files
+     */
+    public List<ChartMetrics> countDownloadedFiles() {
+        return mapToChartMetrics(em.createNativeQuery(
+                "SELECT\n" +
```

Comment:
Maybe we could remove these ```\n```-s?

Reply:
They come from copying working sql from pgAdmin to java IDE, but sure.